packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -315,6 +317,58 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>())
     }
 
+    @Test
+    fun fetchSession_returnsSuccess() {
+        val id: AuthSessionResult<String> = AuthSessionResult.success("id")
+        val awsCredentials: AuthSessionResult<AWSCredentials> = AuthSessionResult.success(BasicAWSCredentials("access", "secret"))
+        val userSub: AuthSessionResult<String> = AuthSessionResult.success("sub")
+        val tokens: AuthSessionResult<AWSCognitoUserPoolTokens> = AuthSessionResult.success(AWSCognitoUserPoolTokens("access", "id", "refresh"))
+        val mockSession = AWSCognitoAuthSession(
+        true,
+            id,
+            awsCredentials,
+            userSub,
+            tokens
+        )
+
+        // Arrange
+        doAnswer { invocation: InvocationOnMock ->
+            plugin.prepareCognitoSessionResult(mockResult, mockSession)
+            null as Void?
+        }.`when`(mockAuth).fetchAuthSession(ArgumentMatchers.any<Consumer<AuthSession>>(), ArgumentMatchers.any<Consumer<AuthException>>())
+
+        val data: HashMap<*, *> = hashMapOf(
+                "getAWSCredentials" to true
+        )
+        val arguments: HashMap<String, Any> = hashMapOf("data" to data)
+        val call = MethodCall("fetchAuthSession", arguments)
+        val res = mapOf(
+            "isSignedIn" to true,
+            "identityId" to "id",
+            "userSub" to "sub",
+            "credentials" to mapOf(
+                "value" to mapOf(
+                    "accessKey" to "access",
+                    "secretKey" to "secret"
+                ),
+                "type" to "SUCCESS"
+            ),
+            "tokens" to mapOf(
+                "value" to mapOf(
+                    "accessToken" to "access",
+                    "idToken" to "id",
+                    "refreshToken" to "refresh"
+                ),
+                "type" to "SUCCESS"
+            )
+        )
+        // Act
+        plugin.onMethodCall(call, mockResult)
+
+        // Assert
+        verify(mockResult, times(1)).success(res);
```

Comment:
`times(1)` is the default behavior, when left unspecified. You can equivalently write this as:

```suggestion
        verify(mockResult).success(res);
```

Reply:
@jamesonwilliams I prefer to specifiy the value rather than rely on knowledge of default behavior, given that this project involves so may different platforms, testing frameworks, etc.